utils.js

```diff
@@ -74,6 +76,7 @@ function sendPing(key) {
     version: 1,
     tab_center_tabs_on_top: prefs.prefs.opentabstop,
     tab_center_show_thumbnails: prefs.prefs.largetabs,
+    tab_center_window_id: this.window.VerticalTabsWindowId,
```

Comment:
I think we should add the current toggled on/off state here, too.
(Oh, and I would be very surprised if `this.window` gave us useful information!  I think we need to pass the window in to sendPing.)